src/@types/app.ts

```diff
@@ -17,6 +17,8 @@ export type JobCounts = Record<JobStatus, number>
 
 export interface QueueAdapter {
   readonly client: Promise<Redis.Redis>
+  readonly readOnlyMode: boolean
```

Comment:
Would it make sense to mark this as an optional parameter like
```ts
  readonly readOnlyMode?: boolean
```

Reply:
This is a class property definition, the value will be always boolean
https://github.com/vcapretz/bull-board/pull/220/files#diff-0acb621af47463a554bde3931cee27d24841eb1ceb1c3efc3e789ea2f4779045R18